_glossary/instance.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Instance
+type: markdown
+---
+{% include home.html %}
+
+An instance represents a chunk of Mixer data that is produced by inspecting a
```

Comment:
maybe this should be "Mixer Instance" as instance seems quite generic (a pod could be an instance of a service for instance (ah!))